templates/support/community-support.html

```diff
@@ -1,55 +1,68 @@
-    </div>
-    <div class="col-4 p-card">
-      <h3><a href="https://help.ubuntu.com/">Read the docs</a></h3>
-      <p>If you're stuck on a problem, someone else has probably encountered it too. Take a look at the <a href="https://help.ubuntu.com">official documentation</a>  or our other <a href="https://docs.ubuntu.com">product docs</a>.</p>
-    </div>
-    <div class="col-4 p-card">
-      <h3><a href="https://answers.launchpad.net/ubuntu">Technical answers system</a></h3>
-      <p>Use Launchpad to add your support question. Keep your query active until you get an answer or browse and search historical questions and answers.</p>
+  <div class="p-strip is-deep">
+    <div class="row u-equal-height">
+      <div class="col-6 p-card">
+        <h3>
+          <a href="/community/support">Find community support</a>
+        </h3>
+        <p>
+          <a href="https://askubuntu.com/">Ask Ubuntu</a> and <a href="https://ubuntuforums.org/">Ubuntu Forums</a> are great sources of answers to common questions about installing, troubleshooting and optimising Ubuntu. For more info on support forums, including IRC channels, see <a href="/community/support">our community docs</a>.
+        </p>
+      </div>
+      <div class="col-6 p-card">
+        <h3>
+          <a href="https://help.ubuntu.com/">Read the docs</a>
+        </h3>
+        <p>
+          If you're stuck on a problem, someone else has probably encountered it too. Take a look at the <a href="https://help.ubuntu.com">official documentation</a>  or our other <a href="https://docs.ubuntu.com">product docs</a>.
+        </p>
+      </div>
     </div>
   </div>
-</div>
-
-{% with first_item="_support_landscape", second_item="_support_contact_us", third_item="_further_reading" %}{% include "shared/contextual_footers/_contextual_footer.html" %}{% endwith %}
-
-<!-- Set default Marketo information for contact form below-->
-<div class="u-hide" id="contact-form-container" data-form-location="/shared/forms/interactive/support" data-form-id="1240" data-lp-id="2065" data-return-url="https://ubuntu.com/support/thank-you" data-lp-url="https://pages.ubuntu.com/things-contact-us.html">
-</div>
 
+  {% with first_item="_support_landscape", second_item="_support_contact_us", third_item="_further_reading" %}
+    {% include "shared/contextual_footers/_contextual_footer.html" %}
+  {% endwith %}
 
+  <!-- Set default Marketo information for contact form below-->
+  <div class="u-hide"
+       id="contact-form-container"
+       data-form-location="/shared/forms/interactive/support"
+       data-form-id="1240"
+       data-lp-id="2065"
+       data-return-url="https://ubuntu.com/support/thank-you"
+       data-lp-url="https://pages.ubuntu.com/things-contact-us.html"></div>
 
 {% endblock content %}
-{% block footer_extra %}{{ marketo }}{% endblock footer_extra %}
+
+{% block footer_extra %}
+  {{ marketo }}
+{% endblock footer_extra %}
```

Comment:
Do we need this? I have never seen this before

Reply:
Doesn't look like we do, thanks for catching